plugins/command-line/prism-command-line.js

```diff
@@ -79,7 +79,7 @@ Prism.hooks.add('before-insert', function (env) {
 
 	// Reinsert the output lines into the highlighted code. -- cwells
 	var codeLines = env.highlightedCode.split('\n');
-	for (var i = 0; i < commandLine.outputLines.length; i++) {
+	for (var i = 0; i < (commandLine.outputLines.length || 0); i++) {
```

Comment:
If `commandLine` empty, `commandLine.outputLines` will be `undefined` meaning that `commandLine.outputLines.length` will throw a reference error.

```suggestion
	for (var i = 0, l = (commandLine.outputLines || []).length; i < l; i++) {
```